scripts/srpm/fapolicy-analyzer.spec

```diff
@@ -10,21 +10,24 @@ Source0:       fapolicy-analyzer.tar.gz
 # reference: https://bugzilla.redhat.com/show_bug.cgi?id=2124697#c5
 Source1:       vendor-rs.tar.gz
 
+# this tarball contains documentation used to generate help docs
+Source2:       vendor-docs.tar.gz
+
 # on copr the source containter is never el
 # we check for low fc version here to remedy that
 %if 0%{?rhel} || 0%{?fedora} < 37
-Source2:       %{pypi_source setuptools-rust 1.1.2}
-Source3:       %{pypi_source pip 21.3.1}
-Source4:       %{pypi_source setuptools 59.6.0}
-Source5:       %{pypi_source wheel 0.37.0}
-Source6:       %{pypi_source setuptools_scm 6.4.2}
-Source7:       %{pypi_source semantic_version 2.8.2}
-Source8:       %{pypi_source packaging 21.3}
-Source9:       %{pypi_source pyparsing 2.1.0}
-Source10:      %{pypi_source tomli 1.2.3}
-Source11:      %{pypi_source flit_core 3.7.1}
-Source12:      %{pypi_source typing_extensions 3.7.4.3}
-Source13:      https://files.pythonhosted.org/packages/source/p/pytz/pytz-2017.2.zip
+Source10:       %{pypi_source setuptools-rust 1.1.2}
```

Comment:
Bumped these up by several numbers to avoid bumping them all again every time a lower numbered source is added.  Might never happen again, but it could.